pages/mainnet.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <main class="container color-primary">
+    <AppHeader />
+    <div class="content_container">
+      <div class="center">
+        <h1 class="heading font-sans">Mainnet Launching Soon</h1>
+        <p class="heading_caption color-secondary font-serif">
+          Join 50+ Web3 projects and 1000s of devs building scalable dApps using
+          Arcana’s Auth, Store and Access
+        </p>
+      </div>
+      <div class="center">
+        <div v-if="loading" class="font-serif">Please Wait...</div>
+        <div v-else class="center">
+          <form v-if="!success" class="email_input_container font-serif">
+            <input
+              v-model="email"
+              type="email"
+              placeholder="Your email address"
+              class="email_input_field"
+            />
+            <button
+              class="email_input_button cursor-pointer"
+              @click.prevent="signup"
+            >
+              Notify Me
+            </button>
+          </form>
+          <div v-else class="email_success_message font-serif">
+            <p>AWESOME! YOU’RE ON THE LIST!</p>
+          </div>
+          <v-text class="subscription-error-message">
+            {{ message }}
+          </v-text>
+        </div>
+        <p class="email_input_caption color-secondary font-serif">
+          Provide your email address to stay updated, get giveaways, NFT drops
+          and more. We won’t spam, promise!
+        </p>
+      </div>
+    </div>
+    <MainnetFooter />
+  </main>
+</template>
+
+<script>
+import { groupTypes, subscribe } from '~/services/mailchimp'
+
+export default {
+  layout: 'default',
+  data() {
+    return {
+      email: '',
+      success: false,
+      message: '',
+      loading: false,
+    }
+  },
+  head() {
+    return {
+      title: 'Mainnet - Arcana Network',
+    }
+  },
+  methods: {
+    async signup() {
+      this.loading = true
+      if (this.email.trim()) {
+        try {
+          await subscribe({
+            email: this.email.trim(),
+            groups: [groupTypes.MAINNET],
```

Comment:
We need both Mainnet and Newsletter. Reconfirm with Ronak

```js
groups: [groupTypes.MAINNET, groupTypes.NEWSLETTER]
```

Reply:
https://github.com/arcana-network/website/pull/139/commits/cd5bbb6963f0362461b352de81b63010aa4fa79c